workloads/logging/deploy_logging_stack.sh

```diff
@@ -44,7 +44,7 @@ function cleanup() {
   while [[ $( oc get projects | grep -w "openshift-logging\|openshift-operators-redhat") ]]; do
     sleep 5
     wait_time=$((wait_time+5))
-    if [[ $wait_time -ge $TIMEOUT ]]; then
+    if [[ "${wait_time}" -ge "${TIMEOUT}" ]]; then
```

Comment:
Can we can reduce this whole function into:
```shell
cleanup() {
  oc delete --wait=true project openshift-logging openshift-operators-redhat --ignore-not-found --timeout=${TIMEOUT}
}

Reply:
rewritted cleanup function to use the --wait